README.md

```diff
@@ -0,0 +1,15 @@
+# dvbtee-express
+This is a basic example of how to use node-dvbtee with udp socket.
+Thanks a lot to @mirmilad
```

Comment:
not going to thank me ?  O:-)  I think we can omit this line from the `README.md` ... If you want to instead create a `CONTRIBUTORS` file, include mirmilad, yourself and I.